src/app/(auth)/sign-up/signUpValidationSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const SignUpValidationSchema = z
+  .object({
```

Comment:
Should any of that be required?

Reply:
Thank you for your suggestion!  In Zod, fields are required by default unless explicitly marked as optional. (z.optional() or z.nullable()).